### DISCUSSION TITLE
Default open_ducument path

### DISCUSSION THREAD

**Discussion by x61** [ahrm/sioyek#626] [General]
When I pressed `o`, I am presented with the root directory to select a file to open. I would like to know if there is a way to specify the default directory to open when I press `o` (`open_document`).

**Comment by ahrm** [comment]
Added a `default_open_file_path` config option in b34d446c36e39910bd809cf079a56f5fba32171f.

**Reply by x61** [reply]
Thank you. Is this feature available in the current version 2.0 or this is for a future update?

**Reply by ahrm** [reply]
It is not available in 2.0 but there are experimental builds here: https://github.com/hexomancer/sioyek/releases/tag/v0.31.429.

**Reply by x61** [reply]
Thank you. I can wait for it in the next update.
